src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -84,9 +87,9 @@ module argument_mod
   integer, public, parameter :: ANY_SPACE_8  = 208
   integer, public, parameter :: ANY_SPACE_9  = 209
   integer, public, parameter :: ANY_SPACE_10 = 210
-  ! Distinct any_discontinuous_space IDs. Separate IDs required
-  ! as we may have groups of fields that must be on the same space
-  ! within a kernel
+  ! IDs for any W2-type space regardless of continuity
```

Comment:
Please clarify that this is not actually for *any* W2 space.

Reply:
Clarified and added a reference to #540.